### DISCUSSION TITLE
GitHub Copilot chat broken in local devcontainer and in Codespaces

### DISCUSSION THREAD

**Discussion by andraz** [community/community#109404] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Running in a local devcontainer Copilot works ok in the editor, but all chat functionalities including the right-click menu are broken. The "hi" i wrote in the chat input simply can not get submitted.

![image](https://github.com/community/community/assets/8480981/12aea715-a50a-4f70-aa71-e625346d6915)

Running the same devcontainer configuration via github Codespaces in the web browser gets past the problem above, but it breaks on the second answer with a map error:

![image](https://github.com/community/community/assets/8480981/6474fb32-5956-403c-bcbe-fe222f310ec1)

We are using the Enterprise version with beta features enabled on our account.

**Comment by andraz** [comment]
Interestingly, combining both, locally installed VSCode executing (the same as in the browser) Codespaces environment, works ok:
![image](https://github.com/community/community/assets/8480981/b60bdb34-5583-4571-a97a-d3e4d34ce549)

But I wish to have it running in local devcontainer, not in local UI of a cloud Codespace 🙂

**Comment by gheiythtba** [comment]
Ensure you are utilizing the most recent version of GitHub Copilot and maintaining an up-to-date development environment cz u know updates may encompass bug fixes and enhancements that contribute to overall stability and performance or review your local devcontainer configuration or Codespaces setup meticulously and onfirm that all essential dependencies and configurations are accurately configured in alignment with the guidelines provided in the GitHub Copilot documentation.

**Reply by andraz** [reply]
All versions are latest, some might be too-latest as they're pre-release.

**Comment by andraz** [comment]
Changing versions from release to pre-release helped in some situations, it broke when automatically installing the extension through configuration of the devcontainer configuration (it installed release, did not work until switched to pre-release in code insiders).

source: https://github.com/microsoft/vscode-copilot-release/issues/436
